frontend/text.js

```diff
@@ -169,7 +169,7 @@ for (let method of ['concat', 'every', 'filter', 'find', 'findIndex', 'forEach',
                     'slice', 'some', 'toLocaleString']) {
   Text.prototype[method] = function (...args) {
     const array = [...this]
-    return array[method].call(array, ...args)
+    return array[method].call(array, ...args) // eslint-disable-line
```

Comment:
These are more of the returns that appear to be non-standard. This is probably undesirable behaviour but is user-facing so we should consider a change more carefully (and try to fix it before 1.0 if a fix is required.)